src/migrations/r-migration-commands/m129_130_flatten_product_card_add_to_cart_cta_button.ts

```diff
@@ -0,0 +1,39 @@
+import { RecordNode, r, rtp, RT, en } from "../../r";
+import { IOrder } from "../IOrder";
+
+/** 
+ * Flattens Product card Add to cart CTA button.
+ * show_add_to_cart]: {
+    show_cart_btn: false,
+    cart_btn_text: "Add to cart",
+    cart_btn_link: "#",
+  } gets converted to
+  show_add_to_cart_button, add_to_cart_button_text, add_to_cart_button_link
+ */
+
+class Migration implements IOrder {
+  execute (projectJson: unknown) {
+    const pJson = projectJson as RecordNode<RT.project>;
+    const projectF = r.record(pJson);
+
+    const productCards = projectF.getAllDeepChildrenWithFilter(RT.element, el => el.props.element_type === en.ElementType.product_card);
+
+    for (const card of productCards) {
+      const elementF = r.element(card);
+      const showAddToCartProperties = elementF.getValueOrDefault(rtp.element.show_add_to_cart) as any;
```

Comment:
After deleting `ElementProperty.show_add_to_cart` call this like so:
`const showAddToCartProperties = elementF.getValueOrDefault(rtp.element.show_add_to_cart as any);`